projects/step-frontend/src/lib/modules/scheduler/components/scheduler-configuration/scheduler-configuration.component.html

```diff
@@ -2,10 +2,10 @@
   <mat-card-content>
     <step-form-field>
       <step-label>
-        Enable scheduler
```

Comment:
This should be kept unchanged. This is still the scheduler who is enabled / disabled